components/Button/button.css

```diff
@@ -0,0 +1,199 @@
+:root {
+
+    --primary-color: #38bdf8;
+    --primary-font: white;
+
+    --secondary-color: #f3f4f6;
+    --secondary-font: #38bdf8;
+
+    --disabled-font: darkgray;
+    --disabled-color: #d1d5db;
+
+    --outlined-bg-color: white;
+    --solid-border-color: white;
+
+
+    --ot-primary-font-color: #38bdf8;
+    --ot-primary-border-color: #38bdf8;
+
+    --ot-secondary-font-color: #9ca3af;
+    --ot-secondary-border-color: #9ca3af;
+
+    --primary-text:#38bdf8;
+    --secondary-text:#9ca3af;
+
+    --primary-hover-color:#7dd3fc;
+    --seconadry-hover-color:#f3f4f6;
+
+}
+
+
+
+.button-container.solid  {
+    display: grid;
+    grid-gap: 1rem;
```

Comment:
grid-gap has been deprecated by Mozilla and is renamed by gap. It might be working because of backward compatibility but it is good practice to use gap from now.

Reply:
Awesome naman i did not knew that i will also keep that in mind

Reply:
Sure. I will keep that in mind.